common/src/dyn_regs.C

```diff
@@ -153,38 +153,44 @@ std::string MachRegister::name() const {
 
     std::string ret;
     if (iter != names()->end()) {
-        signed int arch = reg & 0xff000000;
-        if ( arch == Arch_amdgpu_vega){
+        if ( getArchitecture() == Arch_amdgpu_vega){
             signed int category = reg & 0x00ff0000;
             signed int base_val = reg & 0x000000ff;
             switch(category){
                 case amdgpu_vega::SGPR_VEC2:
+                    base_val -=  (amdgpu_vega::sgpr_vec2_0&0xff);
```

Comment:
Did you mean to include these `base_val` calculation changes?

Reply:
I thought these changes are included in the #948 , which is a pull request from the same branch as for this pull request,

https://github.com/dyninst/dyninst/pull/948/files/53a12aaeab008e36dd9a029229a2d0d599345d4b#diff-b7d9323c9547d36fd98664d3f98917e11290daaacbef0742396b393df003e52d

Reply:
Ah, you'll need to rebase this branch onto the head of master. The pull requests generate separate commit IDs, so github thinks yours is a different branch.

Reply:
Fetched and Force pushed, so this should have been resolved.